doc/onboarding-extras.md

```diff
@@ -25,6 +25,7 @@
 | `src/async-wrap.*` | @nodejs/async_hooks |
 | `src/node_api.*` | @nodejs/n-api |
 | `src/node_crypto.*` | @nodejs/crypto |
+| `src/inspector_*`, `lib/inspector.js` | @nodejs/v8-inspector |
```

Comment:
Actually, looking more closely, it might make sense to change it so `lib/inspector.js` comes first in the line befor `src/inspector_*`. That would be consistent with elsewhere in the doc. (I guess everything defaults alphabeitcal?) And then move the line to the corresponding location in the list. 

Sorry for the nit-picking. Docs are like that. :-|